packages/destination-actions/src/destinations/talon-one/trackEvent/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Track Event',
+  description: 'Track Event in Talon.One',
+  fields: {
+    customer_profile_id: {
+      label: 'customer_profile_id',
+      description: 'ID of the customer profile as used in Talon.One',
+      type: 'string',
+      required: true
+    },
+    event_type: {
+      label: 'event_type',
+      description: 'Name of the custom attribute of type `event`',
+      type: 'string',
+      required: true
+    },
+    type: {
+      label: 'type',
+      description: 'Type of event. Can be only `string`, `time`, `number`, `boolean`, `location`',
+      type: 'string',
+      required: true
+    },
+    event_attributes: {
+      label: 'type',
+      description: 'Arbitrary additional JSON data associated with the event',
+      type: 'object',
+      required: false
+    }
+  },
```

Comment:
@bolatkhanpm track event

Reply:
This records a custom event in Talon.One.
Unique identifier of the customer profile associated to the event.
It's just the name of your event.